dynamiqs/integrators/mesolve/diffrax_integrator.py

```diff
@@ -39,11 +37,13 @@ def terms(self) -> dx.AbstractTerm:
         # induced on the dynamics.
 
         def vector_field(t, y, _):  # noqa: ANN001, ANN202
-            Ls = stack([L(t) for L in self.Ls])
-            Lsd = dag(Ls)
-            LdL = (Lsd @ Ls).sum(0)
-            tmp = (-1j * self.H(t) - 0.5 * LdL) @ y + 0.5 * (Ls @ y @ Lsd).sum(0)
-            return tmp + dag(tmp)
+            tmp = -1j * self.H(t) @ y
+            for L in self.Ls:
+                Lt = L(t)
+                Ltdag = Lt.dag()
+                Lt_y = Lt @ y
+                tmp += 0.5 * (Lt_y @ Ltdag - Ltdag @ Lt_y)
```

Comment:
I don't think it'll play nice with JIT-ting. `tmp = tmp + ...` is safer, or we should try it